packages/fxa-settings/src/components/ContentSkip/index.test.tsx

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import ContentSkip from '.';
+
+describe('ContentSkip', () => {
+  it('links to #main with expected text', () => {
+    render(<ContentSkip />);
+
+    expect(screen.getByTestId('content-skip')).toHaveTextContent(
+      'Skip to content'
+    );
```

Comment:
Theoretically this could simply be `expect(screen.getByText('Skip to content')).toBeInTheDocument();`

Reply:
Well, maybe it's not simpler. idk

Reply:
Eh yeah, it'll error either way if it's not in the doc and I had to attach a `test-id` anyways for the first one.